packages/legend/README.md

```diff
@@ -0,0 +1,42 @@
+# `legend`
+
+> A library to get legend graphics from the map-context.
+
+## Installation
+
+To install the package, use npm:
+
+```sh
+npm install @geospatial-sdk/legend
+```
+
+## Usage
+
+```typescript
+import { createLegendFromLayer } from "@geospatial-sdk/legend";
+
+const layer = {
+  type: "wms",
+  url: "https://example.com/wms",
+  name: "test-layer",
+};
+
+createLegendFromLayer(layer).then((legendDiv) => {
+  document.body.appendChild(legendDiv);
+});
+```
+
+## API Documentation
```

Comment:
The API documentation should be written in the code as jsdoc annotations; there's an automatic website generated from it :) you can try it using `npm run docs:dev`